scripts/ci/run-jenkins.sh

```diff
@@ -262,14 +262,4 @@ if [[ ${CI_TAGS} == *'checked-all'* ]]; then export MONO_CHECK_MODE=all; fi
 
 export MONO_ENV_OPTIONS="$MONO_ENV_OPTIONS $MONO_TEST_ENV_OPTIONS"
 
-if   [[ ${CI_TAGS} == *'acceptance-tests'* ]];         then ${MONO_REPO_ROOT}/scripts/ci/run-test-acceptance-tests.sh;
-elif [[ ${CI_TAGS} == *'profiler-stress-tests'* ]];    then ${MONO_REPO_ROOT}/scripts/ci/run-test-profiler-stress-tests.sh;
-elif [[ ${CI_TAGS} == *'stress-tests'* ]];             then ${MONO_REPO_ROOT}/scripts/ci/run-test-stress-tests.sh;
-elif [[ ${CI_TAGS} == *'interpreter'* ]];              then ${MONO_REPO_ROOT}/scripts/ci/run-test-interpreter.sh;
-elif [[ ${CI_TAGS} == *'mcs-compiler'* ]];             then ${MONO_REPO_ROOT}/scripts/ci/run-test-mcs.sh;
-elif [[ ${CI_TAGS} == *'mac-sdk'* ]];                  then ${MONO_REPO_ROOT}/scripts/ci/run-test-mac-sdk.sh;
-elif [[ ${CI_TAGS} == *'no-tests'* ]];                 then exit 0;
-else make check-ci;
-fi
-
-${MONO_REPO_ROOT}/scripts/ci/run-upload-sentry.sh
+${MONO_REPO_ROOT}/scripts/ci/run-test-microbench.sh;
```

Comment:
By removing all of these, you are basically making it only possible to run the microbenchmarks and nothing else.

Reply:
The title of the commit is `[do-not-merge] Make CI run only the microbench on every lane` for that reason. I wanted to see CI test only the microbench for now.